src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,70 @@
+import classNames from 'classnames';
+import React from 'react';
+import { FilterParam } from '../../types/FilterParam';
+import { Todo } from '../../types/Todo';
+import {
+  checkCompletedTodo,
+  counterOfActiveTodos,
+} from '../../Helpers/helpers';
+
+type Props = {
+  todos: Todo[],
+  filterType: FilterParam,
+  setFilterType: (type: FilterParam) => void,
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  filterType,
+  setFilterType,
+}) => {
+  const changedFilter = (event: React.MouseEvent<HTMLAnchorElement>) => {
```

Comment:
```suggestion
  const handleFilterChange = (event: React.MouseEvent<HTMLAnchorElement>) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364